hub/config/src/main/java/uk/gov/ida/hub/config/resources/CertificatesResource.java

```diff
@@ -161,4 +161,17 @@ private void addCertificateHealthCheckDtos(
                     configuration.getCertificateWarningPeriod()));
         }
     }
+
+    private Collection<Certificate> getCertificates(Set<TransactionConfig> transactionConfigs, Set<MatchingServiceConfig> matchingServiceConfigs) {
+        return Stream.concat(transactionConfigs.stream(), matchingServiceConfigs.stream())
+                .flatMap(this::getCertificates)
+                .collect(Collectors.toSet());
+    }
+
+    private Stream<Certificate> getCertificates(CertificateConfigurable config){
```

Comment:
Is this duplicate code from the config object?

Reply:
Yup,  forgot to clear this instance up.